s2cloudless/sentinelhub_masking.py

```diff
@@ -201,45 +202,51 @@ def _download_bands_and_valid_data_mask(self):
         norm_factors = [response["userdata.json"]["norm_factor"] for response in responses]
         del responses
 
+        if data is None:
+            raise ValueError("Data should not be None.")
```

Comment:
how is data `None` here? dont you define it in L201?

Reply:
we do, removed